src/Orchestration/Adapter/K8s.php

```
@@ -0,0 +1,739 @@
+            $pod->update();
+
+            return true;
+        } catch (KubernetesAPIException $e) {
+            throw new Orchestration("Failed to connect network: {$e->getMessage()}");
+        }
+    }
+
+    /**
+     * Disconnect a container from a network (Remove label from pod)
+     */
+    public function networkDisconnect(string $container, string $network, bool $force = false): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$network'. (undefined)

(UnusedFormalParameter)